pages/linux/lvcreate.md

```diff
@@ -0,0 +1,19 @@
+# lvcreate
+
+> Creates a logical volume in an existing volume group.
+
+- Create a logical volume 10 gigabytes in size in the volume group vg1:
+
+`lvcreate -L {{10G}} {{vg1}}`
+
+- Create a 1500 megabyte linear logical volume named mylv in the volume group vg1:
+
+`lvcreate -L {{1500}} -n {{mylv}} {{vg1}}`
+
+- Create a logical volume called mylv that uses 60% of the total space in volume group vg1:
+
+`lvcreate -l {{60%VG}} -n {{mylv}} {{vg1}}`
```

Comment:
Hello @agnivade, `VG` is part of the parameter, it can be `VG` or `FREE` for `-l` option